tinygrad/state.py

```diff
@@ -80,7 +80,7 @@ def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, bac
 
     return ret.reshape(size)
 
-  intercept = {"HalfStorage": dtypes.float16, "FloatStorage": dtypes.float32, "IntStorage": dtypes.int32, "LongStorage": dtypes.int64, "_rebuild_tensor_v2": _rebuild_tensor_v2}
+  intercept = {"HalfStorage": dtypes.float16, "FloatStorage": dtypes.float32, "BFloat16Storage": dtypes.float16, "IntStorage": dtypes.int32, "LongStorage": dtypes.int64, "_rebuild_tensor_v2": _rebuild_tensor_v2}
```

Comment:
bfloat16 is 8bits exponent 7 bits fraction while float16 is 5bits exponent 10 bits fraction. Can you verify that this is not causing a accuracy loss in the weight values?

Reply:
It does not work at all unfortunately. Please check my comment above: https://github.com/tinygrad/tinygrad/pull/1284#discussion_r1268666825